task03-one-piece-website/onepiece.html

```diff
@@ -0,0 +1,192 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://file.myfontastic.com/KqubBRuJG8iZrRQgJ3mBeb/icons.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;400;700&family=Roboto:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../task03-one-piece-website/css/onepiece.css">
+    <script src="onepiece.js"></script>
+    <title>One Piece</title>
+</head>
+<body>
+    <header>
+          <nav class="navbar fixed-top navbar-expand-xl navbar-light bg-white py-4 justify-content-center ">
+            <button  class="navbar-toggler burger-menu" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
+              <span id="burger-icon" class="icon-align-left"></span>
+              <span id="burger-icon" class="icon-times"></span>
+            </button>
+            <div class="logo">
+              <img class="one-piece-logo"src="../images/one-piece-logo.png" alt="onepiece">
+            </div>
+            <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+              <ul class="navbar-nav">
+                <li class="nav-item">
+                  <a class="nav-link" href="/">Home</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="/characters">Characters</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="/islands">Islands</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/mist-objects">Mist Objects</a>
+                </li>
+              </ul>
+              <div class="flags-container">
+                <div class="flag-item" >
+                    <p class="flag-text">English</p>
+                    <img class="flag-img" src="../images/usa-flag.png">
+                </div>
+                <div class="flag-item" >
+                    <p class="flag-text">Spanish</p>
+                    <img class="flag-img" src="../images/spain-flag.png">
+                </div> 
+              </div> 
+            </div>
+          </nav>
+    </header>
+    <div class="main-photo-info-container">
+      <div class="one-piece-container">
+      <img class="img-one-piece-group" src="../images/one-piece-group.png">
+      <img class="img-second-logo" id="img-second-logo" src="../images/one-piece-logo.png">
```

Comment:
What is the use of this second logo? Is not visible when opening the whole paragraph

Reply:
it's the One Piece logo that is above the sinopsis container, it appears when you click the "Read All":)